x-pack/plugins/synthetics/common/constants/rest_api.ts

```diff
@@ -45,6 +45,7 @@ export enum API_URLS {
   TRIGGER_MONITOR = '/internal/uptime/service/monitors/trigger',
   SERVICE_ALLOWED = '/internal/uptime/service/allowed',
   SYNTHETICS_APIKEY = '/internal/uptime/service/api_key',
+  SYNTHETICS_OVERVIEW = '/internal/synthetics/overview',
```

Comment:
Please move this to https://github.com/elastic/kibana/blob/main/x-pack/plugins/synthetics/common/constants/synthetics/rest_api.ts

It will help in cleanup